.github/ISSUE_TEMPLATE/technical.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Tech
+about: A task, enhancement idea, or bug report related to the technical aspects of the project.
+title: A concise description of the issue
+labels: ''
+assignees: ''
+
+---
+**Description**
+What needs to be improved? What should be done? Has there been any related work before? Any relevant links or material?
+
+**Organization**
+If you have the appropriate repo permissions, please don't forget:
+- Properly label your issue and place it in the appropriate project.
+- If you assign yourself or someone else, please ensure the project status is updated to "Assigned".
```

Comment:
Can this step be automated?

Reply:
Eventually, yes. It's in our automation plan. But we don't know how long these specific actions will take to implement, and I think it's good to have issue templates ready now for organization/ease of use.